pandas/tools/tests/test_tile.py

```diff
@@ -313,6 +313,18 @@ def test_datetime_cut(self):
         result, bins = cut(data, 3, retbins=True)
         tm.assert_series_equal(Series(result), expected)
 
+    def test_datetime_bin(self):
+        data = [np.datetime64('2012-12-13'), np.datetime64('2012-12-15')]
+        bins = [np.datetime64('2012-12-12'), np.datetime64('2012-12-14'),
```

Comment:
@jreback made the changes, i added a new method in tile.py to handle the time type bins